src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">{`${person.age === undefined ? '' : ` I am ${person.age}`}`}</p>
```

Comment:
It's not a very good idea to check clearly on undefined, it's better to check on true
```suggestion
    {person.age && (
      <p>...</p>
    )}
```